.azure-pipelines-templates/matrix.yml

```diff
@@ -1,7 +1,7 @@
 parameters:
   consensus: ['CFT', 'BFT']
   target: ['NoSGX', 'SGX']
-  protocol: ['FTCP', 'HTTP']
+  protocol: [HTTP']
```

Comment:
Do we still need this at all?

Reply:
Probably not, WS will be in addition to, not a separate thing. Still, it doesn't hurt to keep it for now, until we're done with all the cleanup that has measurable benefits (ie. shorter builds, accurate doc, easier refactoring for RpcContext).

Reply:
Ah, and the thing with taking it off it will confuse CI Metrics somewhat of course, since the metrics names are composed that way etc.